e2e/types/index.ts

```diff
@@ -0,0 +1,218 @@
+import { type OpenmrsResource } from '@openmrs/esm-framework';
+
+export type PatientRegistrationSex = 'male' | 'female' | 'other' | 'unknown';
+
+export interface PatientRegistrationFormValues {
+  givenName?: string;
+  middleName?: string;
+  familyName?: string;
+  sex?: PatientRegistrationSex;
+  birthdate?: {
+    day: string;
+    month: string;
+    year: string;
+  };
+  postalCode?: string;
+  address1?: string;
+  address2?: string;
+  country?: string;
+  countyDistrict?: string;
+  stateProvince?: string;
+  cityVillage?: string;
+  phone?: string;
+  email?: string;
+}
```

Comment:
I also don't think we need this. Since this is per-repo, it's best to keep it stripped down to the things we're using.